packages/jest-types/__typetests__/each.test.ts

```diff
@@ -6,10 +6,11 @@
  */
 
 import {expectError, expectType} from 'tsd-lite';
+import {number} from 'yargs';
```

Comment:
this seems random?

Reply:
Came from some autocompletion. No need of it.